tests/php/ORM/DataQueryTest.php

```diff
@@ -448,7 +450,7 @@ public function testComparisonClauseTextCaseSensitive()
         static::resetDBSchema(true);
     }
 
-    public function testAddToQueryIsCalled()
+    public function testFinalisedQueryCallsAddToQuery()
```

Comment:
Renamed this and the other method below to make it clearer they're testing the results of `finalisedQuery()` and to give me a naming scheme to latch onto with my new test method.